content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -0,0 +1,513 @@
+---
+title: OpenTelemetry Protocol with Apache Arrow in Production
+linkTitle: OpenTelemetry Arrow in Production
+date: 2024-09-13
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Joshua MacDonald](https://github.com/jmacd) (ServiceNow, Inc), [Laurent
+  Querel](https://github.com/lquerel) (F5, Inc)
+draft: true
+issue: 5193
+sig: OpenTelemetry Arrow
+---
+
+## OpenTelemetry Protocol with Apache Arrow in Production
+
+The OpenTelemetry Protocol with Apache Arrow project's Exporter and Receiver
+components for the OpenTelemetry Collector are now included in OpenTelemetry
+Collector-Contrib releases. This is a case study of our experience deploying
+OpenTelemetry Collectors using OTel-Arrow components as the primary ingestion
+path for internal telemetry data at ServiceNow Cloud Observability.
+
+Since the project’s announcement our collaboration with F5, Inc. has focused on
+making the Exporter and Receiver components into a reliable, high-performance
+method for transporting bulk OpenTelemetry data across expensive network links.
+Using these components for our internal telemetry, we observe compression
+factors in the range of 15x to 30x of uncompressed size (i.e., 15 to 30 times
+smaller). In a side-by-side comparison between OpenTelemetry Protocol (“OTLP”)
+and OpenTelemetry Protocol with Apache Arrow (“OTAP”) for similarly configured
+traces pipelines, we observe 30% improvement in compression. Although this study
+specifically focused on traces data, we have observed results for logs and
+metrics signals in production settings too, where OTAP users can expect 50% to
+70% improvement relative to OTLP for similar pipeline configurations.
+
+With our previous experimental results now validated in production, the
+OpenTelemetry Protocol with Apache Arrow Exporter and Receiver are considered
+ready for general use.
+
+## Exporter and Receiver Components
+
+Apache Arrow is a great technology partner for OpenTelemetry, and these projects
```

Comment:
```suggestion
Apache Arrow is a great technology partner for OpenTelemetry, and these projects
```

This sounds like the Apache Arrow and OpenTelemetry have some kind of technology partnership, but it's probably more around the technical compatibility?

Reply:
Thank you. I have rewritten this to say "Apache Arrow is a great technical dependency for OpenTelemetry because these projects..."